Mage.Tests/src/test/java/org/mage/test/cards/abilities/oneshot/sacrifice/IndustrialAdvancementTest.java

```diff
@@ -0,0 +1,51 @@
+package org.mage.test.cards.abilities.oneshot.sacrifice;
```

Comment:
You should move this into the ncc folder rather than the abilities folder. The aiblities folder should be for common abilities rather than for one-off implementations for a card